aes.md

```diff
@@ -0,0 +1,146 @@
+If someone declines, or does not respond to the invitation after a few days, invite someone else until 3 reviewers are assigned.
+
+### Does automatic review assignment check if the reviewer is an action editor?	
+We hope that action editors will also be able to serve as reviewers in some cases, as action editors are often the most experienced members of our community who can provide high-quality reviews on some topics. However, when we assign papers we will attempt to balance your load so the overall load of action-edited and reviewed papers is manageable. That being said, you should never be assigned as reviewer of a paper where you are action editor, so if this happens please contact us to notify us of the issue, then unassign yourself and assign a different reviewer.
+
+### Should I assign and invite someone else to review a paper where the reviewer has indicated that they are unavailable or unqualified?	
+Yes, and you can do this by clicking through to "Modify Reviewer Assignments" from the Area Chair Console.
+
+## Compliance and Desk Rejects
+
+### What is the compliance checklist?
+- Is the content of this paper appropriate to a \*ACL venue?
+- Is the paper in English?
+- Does this paper use the ARR template?
+- Does this paper adhere to length constraints for the paper type (long: 8, short: 4, in both cases exclusive of references and appendices)?
+- Is the submission (including paper and any supplemental materials) anonymized?
+- Is this paper not already published elsewhere?
+
+### One of my papers doesn't follow the compliance checklist, what should I do?
+The paper should be desk-rejected, please use the "desk reject" button in OpenReview. If you're not sure about the decision, please get in touch with the editors in chief. 
+
+### One of my assigned submissions is more than four pages but less than eight pages in length. What should I do?
+Treat it as a long paper submission.
+
+### Can the ethics statement extend into a 5th/9th page?
+Yes. 
+
+### How do I actually desk reject a submission?
+In OpenReview, you should see a button labeled "Desk Reject".
+
+## Reviewer Assignment
+
+### In what order are alternative reviewers for a submission displayed (eg alphabetical, by match score, random)?
+You can choose several orders. If you choose "affinity score" they will be sorted by the matching score given by an ML model (specifically, as of September this is the [SPECTRE-MFR model](https://github.com/openreview/openreview-expertise) provided by OpenReview).
+
+### Can we see the number of submissions a reviewer is already assigned?
+Yes, look for the number next to "assignments" by their name. Please do consider this when making new assignments.
+
+### Can we see author information?
+No, we are managing a two-way anonymized process up through action editors. It is not possible for you to assign a reviewer who has a COI that is automatically identifiable.
+
```

Comment:
What about resubmissions? Is it important to assign the original reviewers if possible (or is it OK to replace, for example, a positive but less informative reviewer)? Note that the platform provides access to the new paper as well as authors' summary of changes. Any other special considerations for handling resubmissions?

Reply:
Similarly started a separate issue.